BLOGPOST.md

```diff
@@ -0,0 +1,38 @@
+# Introducing the SAP Cloud SDK for AI (JavaScript/TypeScript) 🎉
+
+We are excited to announce the initial release of the [SAP Cloud SDK for AI](https://github.com/SAP/ai-sdk-js#readme)! It simplifies the integration of generative AI capabilities into your SAP Business Technology Platform (BTP) applications.
+
+The SDK allows you to leverage the SAP AI Core and SAP AI Launchpad, enabling capabilities like templating, grounding, data masking, and more. In this blog post, we’ll introduce you to the key packages and their features.
+
+## Key Features
+### AI Orchestration with `@sap-ai-sdk/orchestration`
+The [@sap-ai-sdk/orchestration](https://github.com/SAP/ai-sdk-js/tree/main/packages/orchestration#readme) package enables generative AI orchestration, allowing you to configure templating, content filtering, and data masking for your applications. With its orchestration service, you can streamline workflows and manage AI models within SAP AI Core.
+
+- **Templating**: Create dynamic prompts with placeholders to customize AI interactions based on user inputs.
+- **Content Filtering**: Apply filters to ensure input and output adhere to content safety guidelines.
+- **Data Masking**: Anonymize sensitive information with data masking features.
```

Comment:
[pp] It would be nice to see some code examples. Consider adding a very minimal usage and linking to readme of the orchestration client for specific code samples related to each module.

Reply:
@jjtang1985 do you think I should put the code examples here, or simply link to our examples, e.g. https://github.com/SAP/ai-sdk-js/blob/main/packages/orchestration/README.md#templating

Reply:
Because I wasn't sure if I keep the blogpost brief and rather not include code examples.
At least we did not do so with our JS SDK Blogposts (which I used as a baseline)